matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/session/content/UploadContentWorker.kt

```diff
@@ -48,10 +48,10 @@ import java.io.File
 import java.util.UUID
 import javax.inject.Inject
 
-private data class NewImageAttributes(
-        val newWidth: Int?,
-        val newHeight: Int?,
-        val newFileSize: Int
+private data class NewAttachmentAttributes(
+        val newWidth: Int? = 0,
+        val newHeight: Int? = 0,
+        var newFileSize: Long
```

Comment:
Why not a val?

Reply:
Because we are changing it (line 209) if api bug occurs.

Reply:
Please make it a val and use copy() if necessary

Reply:
Done.